pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -417,12 +417,12 @@ func doPrepare(ctx *zedmanagerContext,
 	return changed, true
 }
 
-// doActivate - Returns if the status has changed. Doesn't publish any changes.
+// doUpdateActivate - Returns if the status has changed. Doesn't publish any changes.
```

Comment:
I think that `doUpdateActivate` is still confusing - too many verbs :)

Reply:
yeah, probably. How would you name it?

Reply:
I would name it `doActivateApp`. There are also functions like `doInstall`, `doUpdate`, `doPrepare`, etc. in the same file, all changing state of an app instance. So if you are going to rename `doActivate` here, then probably all these other functions should be also renamed accordingly (i.e. adding `App` suffix).